components/org.wso2.carbon.identity.api.server.organization.selfservice/org.wso2.carbon.identity.api.server.organization.selfservice.v1/src/main/java/org/wso2/carbon/identity/api/server/organization/selfservice/v1/core/SelfServiceMgtService.java

```diff
@@ -74,6 +86,8 @@ public class SelfServiceMgtService {
     @Autowired
     private ServerUserStoreService serverUserStoreService;
 
+    public static final String SHARE_WITH_ALL_CHILDREN = "shareWithAllChildren";
```

Comment:
You can use a constant defined in the organization management service level. Better if you can define it in the organization-mgt-core and use it even in the organization management service repo.

Reply:
Addressed with 352a72fadea84c82ec2df6a2cbbf10f382676f30